tests/angular-js/e2e/caseList.js

```diff
@@ -24,13 +24,13 @@
       });
 
       it("should update URL for case filter selections", function() {
-        element(by.linkText("Phone cases")).click();
+        element(by.className('Filters')).element(by.linkText("Phone cases")).click();
```

Comment:
_not a blocker_
Would it be possible to use `by.name` for these filtering elements? E.g. `by.name("filter-phone-cases")`?

Reply:
I don't get any warnings or errors when running `caseList.js` in the develop branch so just wanted to double-check fixing in the right test.

I get this error when running `case.js`:

```
W/element - more than one element found for locator By(css selector, *[name="spoke_to"]) - the first result will be used
```

Reply:
@sldblog I can't use `by.name` without adding more markup to the page. The filters are merely a list of nav links and there is no form.

Reply:
@rowlando thats odd I'm also not getting that error on develop. However I do remember seeing it on the caseLIst and when I investigated it, it made sense that "EOD" was found twice on the page if you logged in as an operator manager. I wonder if my local develop branch was not up to date and perhaps your PR relating to fixing e2e test might have changed the behaviour. You did update protractor and the reporter in that branch.

Anyway, I think this is still good practice to scope the search to a smaller area than a huge global search.